cpp/include/cudf/lists/lists_column_view.hpp

```diff
@@ -87,6 +87,25 @@ class lists_column_view : private column_view {
    * @throw cudf::logic error if this is an empty column
    */
   column_view get_sliced_child(rmm::cuda_stream_view stream) const;
+
+  /**
+   * @brief Return first offset (accounting for column offset)
+   *
+   * @return int32_t const* Pointer to the first offset
+   */
+  using offset_type     = int32_t;
+  using offset_iterator = offset_type const*;
```

Comment:
Move the two `using` lines above the doxygen comment. for the member function.

Reply:
Moved these up to the other `using` lines.